tests/fixture_plugins/demultiplex_fixtures/sample_fixtures.py

```diff
@@ -120,16 +52,45 @@ def hiseq_2500_custom_index_bcl_convert_lims_samples(
 
 
 @pytest.fixture
-def lims_novaseq_samples_raw(lims_novaseq_samples_file: Path) -> list[dict]:
-    """Return a list of raw flow cell samples."""
-    return ReadFile.get_content_from_file(
-        file_format=FileFormat.JSON, file_path=lims_novaseq_samples_file
-    )
+def novaseq_6000_pre_1_5_kits_bcl2fastq_lims_samples(
+    novaseq_6000_pre_1_5_kits_flow_cell: Path,
+) -> list[FlowCellSampleBcl2Fastq]:
+    path = Path(novaseq_6000_pre_1_5_kits_flow_cell, "HLYWYDSXX_bcl2fastq_raw.json")
+    return [FlowCellSampleBcl2Fastq.model_validate(sample) for sample in read_json(path)]
+
+
+@pytest.fixture
+def novaseq_6000_pre_1_5_kits_bcl_convert_lims_samples(
+    novaseq_6000_pre_1_5_kits_flow_cell: Path,
+) -> list[FlowCellSampleBCLConvert]:
+    path = Path(novaseq_6000_pre_1_5_kits_flow_cell, "HLYWYDSXX_bcl_convert_raw.json")
+    return [FlowCellSampleBCLConvert.model_validate(sample) for sample in read_json(path)]
+
+
+@pytest.fixture
+def novaseq_6000_post_1_5_kits_bcl_convert_lims_samples(
+    novaseq_6000_post_1_5_kits_flow_cell: Path,
+) -> list[FlowCellSampleBCLConvert]:
+    path = Path(novaseq_6000_post_1_5_kits_flow_cell, "HK33MDRX3_bcl_convert_raw.json")
+    return [FlowCellSampleBCLConvert.model_validate(sample) for sample in read_json(path)]
```

Comment:
for, someone not super deep in pre, post 1-5 kits I have no idea what this means. Also, how can we re-use `novaseq_6000_pre_1_5_kits_flow_cell` in two fixtures here? 

Would a docstring that provides some context be warranted here? I think someone who sees these fixtures the first time will have a hard time understanding them.

Reply:
The docstring is a great idea! I will add it.

Regarding the picture, it is the path to the fixture data. There, I saved both the bcl_convert and bcl2fastq raw data, so that is why it is reused. However the naming is not good as it give the idea that it is a FlowCellDirectoryData object and not a Path, I will rename it

Reply:
I decided to add a note on the README file of these pre/post flow cells. I think the description is not suitable for this specific docstring. In short, a "pre" flow cell would not have any reverse complemented index in the sample sheet, while the "post", will have the index2 (i5) reversed in the column `index2` of the sample sheet but not on the `OverrideCycles` column

Reply:
did my best with the docstrings, going to resolve this